addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -78,6 +79,7 @@ test("many2one in list view", async () => {
     expect(".o_card_user_infos > a").toHaveText("Mario@partner.com");
     expect(".o_avatar_card_buttons button:eq(0)").toHaveText("Send message");
     await contains(".o_avatar_card_buttons button:eq(0)").click();
+    await waitFor(".o-mail-ChatWindow");
     expect(".o-mail-ChatWindow").toHaveCount(2);
```

Comment:
Would be best to `waitFor(` with "Mario" in header.
Though I'm pretty sure the asynchronous assertion doesn't matter here, because the chat window is already open anyway.